composer/utils/object_store/oci_object_store.py

```diff
@@ -42,29 +62,34 @@ def __init__(
         if self.prefix != '':
             self.prefix += '/'
 
-        if 'OCI_CONFIG_FILE' in os.environ:
-            config = oci.config.from_file(os.environ['OCI_CONFIG_FILE'])
-        else:
-            config = oci.config.from_file()
+        try:
+            if 'OCI_CONFIG_FILE' in os.environ:
+                config = oci.config.from_file(os.environ['OCI_CONFIG_FILE'])
+            else:
+                config = oci.config.from_file()
+
+            self.client = oci.object_storage.ObjectStorageClient(config=config,
+                                                                retry_strategy=oci.retry.DEFAULT_RETRY_STRATEGY)
+        except Exception as e:
+            _reraise_oci_errors(self.get_uri(object_name=''), e)
 
-        self.client = oci.object_storage.ObjectStorageClient(config=config,
-                                                             retry_strategy=oci.retry.DEFAULT_RETRY_STRATEGY)
         self.namespace = self.client.get_namespace().data
         self.upload_manager = oci.object_storage.UploadManager(self.client)
 
     def get_uri(self, object_name: str) -> str:
         return f'oci://{self.bucket}/{object_name}'
 
     def get_object_size(self, object_name: str) -> int:
-        response = self.client.get_object(
-            namespace_name=self.namespace,
-            bucket_name=self.bucket,
-            object_name=object_name,
-        )
-        if response.status == 200:
-            return int(response.data.headers['Content-Length'])
-        else:
-            raise FileNotFoundError(f'Unable to locate oci://{self.bucket}@{self.namespace}/{object_name}')
+        try:
+            response = self.client.get_object(
+                namespace_name=self.namespace,
+                bucket_name=self.bucket,
+                object_name=object_name,
+            )
+        except Exception as e:
+            _reraise_oci_errors(self.get_uri(object_name), e)
+
+        return int(response.data.headers['Content-Length'])
```

Comment:
missing response status 200 check? is that no longer necessary

Reply:
I could add it back in

Reply:
done